widgets/OracleDatastream-cdcSource.json

```diff
@@ -26,8 +26,39 @@
         {
           "name": "region",
           "label": "Region",
-          "widget-type": "textbox",
+          "widget-type": "select",
           "widget-attributes": {
+            "values": [
```

Comment:
This approach is also followed at other places in CDF https://github.com/cdapio/cdap/blob/develop/cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json#L33 primarily because we do not yet have a select widget which can load values from the backend. 

Created https://cdap.atlassian.net/browse/CDAP-19578 to fetch the list from a backend API once a dynamic dropdown is available